SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -294,7 +295,11 @@ void USpatialSender::RetryServerWorkerEntityCreation(Worker_EntityId EntityId, i
 			if (Op.status_code == WORKER_STATUS_CODE_SUCCESS)
 			{
 				Sender->NetDriver->WorkerEntityId = Op.entity_id;
-				Sender->NetDriver->GlobalStateManager->TrySendWorkerReadyToBeginPlay();
+
+				// We claim each server worker entity as a partition so server worker interest which is necessary for getting
+				// interest in the VirtualWorkerTranslator component.
```

Comment:
I couldn't parse this, could you clarify?

Reply:
Re-worded

Reply:
Yeah I actually think this is slightly incorrect, we should have interest in the VWT before this point, will update